tests/test_m2m.py

```diff
@@ -0,0 +1,277 @@
+import unittest
+from lxml import etree
```

Comment:
There should be a space between the first and second import, and then again between the third and fourth import, so it's easy to tell the difference between standard library imports, third party imports, local imports, etc., as per [PEP-8](https://www.python.org/dev/peps/pep-0008/#imports)